Sources/minify/src/JS.php

```diff
@@ -15,7 +15,7 @@
 class JS extends Minify
 {
     /**
-     * Var-matching regex based on http://stackoverflow.com/a/9337047/802993.
+     * Var-matching regex based on https://stackoverflow.com/a/9337047/802993.
```

Comment:
It does, but only through the shorturl redirect that stackoverflow does. I suspect that is something on their end that will change/be fixed someday.  It still produces a valid cert, works and gets you to the same content.